composer.json

```diff
@@ -44,11 +44,12 @@
         "webmozart/assert": "^1.9"
     },
     "require-dev": {
+        "cache/integration-tests": "dev-master",
```

Comment:
Should we pin to a specific commit? 🤔

Reply:
Is renovate able to update commits? Then yes, why not. 👌🏻

Reply:
I think renovate uses composer 's CLI here, not that advanced 😅

Reply:
I removed the requirement from https://github.com/laminas/laminas-cache-storage-adapter-test/releases/tag/2.3.0 so I can drop this requirement here.